release.py

```diff
@@ -90,24 +79,40 @@ def create_and_push_tag(old_version, new_version, release_notes=''):
     push_tag_cmd = 'git push origin %s' % new_version
     run_or_die(push_tag_cmd)
 
-def update_docker_image(new_version, update_latest_tag):
+def push_image(target, version_tag, image_tag):
+    # Note:
+    # The flag "--define=version=..." sets the Docker image tag.
```

Comment:
maybe it would be clearer if we renamed this "version" to "image_tag" in our BUILD files?

Reply:
I agree -- I'll send a follow-up for that because there's a lot of places that need to be updated.